chroma-agent/chroma_agent/action_plugins/manage_node.py

```diff
@@ -14,6 +14,7 @@
 from chroma_agent.chroma_common.lib.agent_rpc import agent_error
 from chroma_agent.chroma_common.lib.agent_rpc import agent_result_ok
 from chroma_agent.lib.agent_startup_functions import agent_daemon_startup_function
+from chroma_agent.lib.agent_teardown_functions import agent_daemon_teardown_function
```

Comment:
So, should this correspond to a `chroma-agent/chroma_agent/lib/agent_teardown_functions.py` file in the source tree?  I'm just not seeing one at all in this branch.